edgelet/edgelet-docker/src/settings.rs

```diff
@@ -310,6 +330,7 @@ mod tests {
     use std::cmp::Ordering;
     use std::fs::File;
     use std::io::prelude::*;
+    use crate::settings::agent_image_resolve;
 
     use serde_json::json;
     use tempdir::TempDir;
```

Comment:
ordering is wrong. should go like below. crates go after all the libraries.     
    
    use std::cmp::
    use std::fs::File;
    use std::io::prelude::*;
   
    use serde_json::json;
    use tempdir::TempDir;

    use crate::settings::agent_image_resolve;

Reply:
Thanks, didn't know about that